test/topology_experimental_raft/test_topology_upgrade.py

```diff
@@ -71,5 +74,10 @@ async def test_topology_upgrade_basic(request, manager: ManagerClient):
     logging.info("Checking consistency of data in system.topology and system.cdc_generations_v3")
     await check_system_topology_and_cdc_generations_v3_consistency(manager, hosts)
 
+    await wait_until_last_generation_is_in_use(cql)
+
+    logging.debug("Sleeping for 1 second to make sure there are writes to the CDC table in the last generation")
```

Comment:
Nit: would prefer INFO, not sure if we enable DEBUG level in our test.py runs by default, and these messages are rare (in fact they happen once in the test), and all the other messages above are also on INFO level.

Reply:
Changed to `info`.